nixos/modules/services/networking/nomad.nix

```diff
@@ -52,12 +52,25 @@ in
       settings = mkOption {
         type = format.type;
         default = {
-          # Agrees with `StateDirectory = "nomad"` set below.
           data_dir = "/var/lib/nomad";
         };
         description = ''
           Configuration for Nomad. See the <link xlink:href="https://www.nomadproject.io/docs/configuration">documentation</link>
           for supported values.
+
+          Notes about `data_dir`:
+
+          If `data_dir` is set to a value other than the default value of
+          `"/var/lib/nomad"` it is the Nomad cluster manager's responsibility
+          to make sure that this directory exists and has the appropriate
+          permissions. One way to ensure this is the case to create the
+          directory and adjust its permissions as needed using the
+          `ExecStartPre` feature of systemd.
+
+          Additionally, if `dropPrivileges` is `true` then `data_dir`
+          **cannot** be customized.  Setting `dropPrivileges` to `true` enables
```

Comment:
This isn't markdown, you need `<emphasis>cannot</emphasis>` I think?

Reply:
Also not sure how to format those inline code blocks. I think it's `<code>true</code>`?

Reply:
`<literal>` might also do the trick here... but I'm terrible with this format so...